azure-sql/database/service-tier-hyperscale.md

```diff
@@ -20,22 +20,22 @@ Azure SQL Database is based on SQL Server Database Engine architecture that is a
 - Business Critical
 - Hyperscale
 
-The Hyperscale service tier is suitable for all workload types. Its cloud native architecture provides independently scalable compute and storage to support the widest variety of traditional and modern applications. Compute and storage resources in Hyperscale substantially exceed the resources available in the General Purpose and Business Critical tiers. 
+The Hyperscale service tier is suitable for all workload types. Its cloud-native architecture provides independently scalable compute and storage to support the widest variety of traditional and modern applications. Compute and storage resources in Hyperscale substantially exceed the resources available in the General Purpose and Business Critical tiers. 
 
 > [!NOTE]
 >
 > - For details on the General Purpose and Business Critical service tiers in the vCore-based purchasing model, see [General Purpose](service-tier-general-purpose.md) and [Business Critical](service-tier-business-critical.md) service tiers. For a comparison of the vCore-based purchasing model with the DTU-based purchasing model, see [Azure SQL Database purchasing models and resources](purchasing-models.md).
-> - The Hyperscale service tier is currently only available for Azure SQL Database, and not Azure SQL Managed Instance.
+> - The Hyperscale service tier is currently only available for Azure SQL Database and not Azure SQL Managed Instance.
 
 ## What are the Hyperscale capabilities
 
 The Hyperscale service tier in Azure SQL Database provides the following additional capabilities:
 
-- Rapid Scale up - you can, in constant time, scale up your compute resources to accommodate heavy workloads when needed, and then scale the compute resources back down when not needed.
+- Rapid Scale up - you can constantly scale up your compute resources to accommodate heavy workloads when needed and then scale the compute resources back down when not needed.
```

Comment:
The original use of "in constant time" is correct. It refers to the fact that each scaling operation takes approximately the same time regardless of database size. That is different from "constantly scale", which refers to scaling operations that occur frequently.